lib/build/build.js

```diff
@@ -85,23 +87,14 @@ class Build {
       }
 
       if (peonConfig.cache && peonConfig.cache.length) {
-        await this._runStep('save cache', () => this._saveCache())
+        await this._runStep('save cache', (updateOutput) =>
+          this._saveCache(updateOutput)
+        )
       }
 
-      await this._runStep('deploy', () => this._deploy())
-
-      let {
-        destination: { absoluteUrl },
-        pathInDestination
-      } = this
+      let deployData = await this._runStep('deploy', () => this._deploy())
```

Comment:
and here is the reason for that change: we need to retrieve data from the deploy step :)